src/lib/redis.ts

```
@@ -21,14 +23,36 @@ redis.on('error', (err) => {
+      '30m': 1800,
+      '1h': 3600,
+      '6h': 21600,
+      '12h': 43200,
+      '24h': 86400,
+      '48h': 172800,
+      '72h': 259200,
+      '1w': 604800,
+      '2w': 1209600,
+      '1mo': 2592000,
+    };
+    expirationInSeconds = expirationLengths[expirationInSeconds];
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)